jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -196,59 +189,99 @@ class JWTTools(
             throw InvalidJWTException("JWT has expired: exp: ${payload.exp}")
         }
 
-        val ddo: DIDDocument = UniversalDID.resolve(payload.iss)
+        val publicKeys = resolveAuthenticator(header.alg, payload.iss)
```

Comment:
resolveAuthenticators should be called with an auth param forwarded from the verify method.
The verify method should use auth=false as default